src/main/java/com/ipiecoles/java/java220/Commercial.java

```diff
@@ -0,0 +1,93 @@
+package com.ipiecoles.java.java220;
+
+import org.joda.time.LocalDate;
+
+public class Commercial extends Employe {
+    private Double caAnnuel;
+    private Integer performance;
+
+    public Commercial(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Double caAnnuel) {
+        super(nom, prenom, matricule, dateEmbauche, salaire);
+        this.caAnnuel = caAnnuel;
+    }
+
+    public Commercial(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Double caAnnuel, Integer perf) {
+        super(nom, prenom, matricule, dateEmbauche, salaire);
+        this.caAnnuel = caAnnuel;
```

Comment:
Appelle l'autre constructeur qui fait déjà presque tout le boulot
```suggestion
        this(nom, prenom, matricule, dateEmbauche, salaire, caAnnuel);
```